src/components/UserAccess/UserAccessForm/UsernameSection.tsx

```diff
@@ -36,28 +35,25 @@ export const UsernameSection: React.FC<React.PropsWithChildren<Props>> = ({ disa
       setValidating(true);
       setValidHelpText('');
       setError('');
+      // username is mandantory
       if (!username) {
         setValidating(false);
         return;
       }
+      // usename format should be valid
       if (!usernameRegex.test(username)) {
         setValidating(false);
         setError('Invalid username format.');
         return;
       }
-      void validateUsername(username).then((valid) => {
-        setValidating(false);
-        if (valid) {
-          setError('');
-          setValidHelpText('Validated');
-          if (!usernames.includes(username)) {
-            void setValue([...usernames, username]);
-          }
-          setUsername('');
-        } else {
-          setError('Username not found.');
-        }
-      });
+      // for valid username
+      setValidating(false);
```

Comment:
Remove validation from UI. As for RegEx fails we should tell user which characters are allow as an error.